sql/handler.cc

```diff
@@ -5264,7 +5264,8 @@ handler::ha_create(const char *name, TABLE *form, HA_CREATE_INFO *info_arg)
     info_arg->options|= HA_LEX_CREATE_GLOBAL_TMP_TABLE;
   int error= create(name, form, info_arg);
   if (!error &&
-      !(info_arg->options & (HA_LEX_CREATE_TMP_TABLE | HA_CREATE_TMP_ALTER)))
+      !(info_arg->options & (HA_LEX_CREATE_TMP_TABLE | HA_CREATE_TMP_ALTER)) &&
+      !info_arg->is_partition)
```

Comment:
Another question, could we not use the following?:
```c
if (share->db_type() == partition_hton)
```
If this works, it would save adding a new variable.

Reply:
Hey Andrew thank your for your comment. I implemented your suggestion locally and tested it. Unfortunately, using (share->db_type() != partition_hton) as the condition removes all CREATE logs for the table creation. The desired behavior is to retain one CREATE log and remove the duplicates.